src/sentry/runner/commands/run.py

```diff
@@ -412,11 +413,18 @@ def inner(f):
 @run.command("ingest-consumer")
 @log_options()
 @click.option(
+    "consumer_types",
     "--consumer-type",
-    default=None,
-    required=True,
-    help="Specify which type of consumer to create, i.e. from which topic to consume messages.",
-    type=click.Choice(["events", "transactions", "attachments"]),
+    default=[],
+    multiple=True,
+    help="Specify which type of consumer to create, i.e. from which topic to consume messages. By default all ingest-related topics are consumed ",
+    type=click.Choice(ConsumerType.all()),
+)
+@click.option(
+    "--all-consumer-types",
```

Comment:
_nit_: `sentry run ingest-consumer --all` might be nicer since it's evident that you mean consumers (based on the command name).